pkg/api/v1/status/atlasproject.go

```diff
@@ -18,4 +23,29 @@ type AtlasProjectStatus struct {
 	// The ID of the Atlas Project
 	// +optional
 	ID string `json:"id,omitempty"`
+
+	// The list of IP Access List entries that are expired due to 'deleteAfterDate' being less than the current date.
+	// Note, that this field is updated by the Atlas Operator only after specification changes
+	ExpiredIPAccessList []ProjectIPAccessList `json:"expiredIpAccessList,omitempty"`
+}
+
+// Copy of mdbv1.ProjectIPAccessList
+// TODO solve circular dependency (move ProjectIPAccessList to subpackage?)
+
```

Comment:
Let's change these lines to avoid TODO appearing in generated files:
```suggestion
// TODO solve circular dependency (move ProjectIPAccessList to subpackage?)

// ProjectIPAccessList is a copy of mdbv1.ProjectIPAccessList
```